blog/2023/12-10-android-tv-16/index.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Android TV version 0.16
+authors:
+  - nielsvanvelzen
+slug: androidtv-v0.16.0
+tags: [release, android-tv]
+---
+
+It has been a busy year with a lot of work on the Android TV app. After an extended beta period today is finally the day to share this work
+to the public. Read along to learn everything about improved music playback, the new screensaver and more!
+
+\- Niels
+
+{/* truncate */}
+
+## Significant changes
+
+In this post I will focus on a few of the popular and significant changes. As always a lot more changes were made both visible and
+invisible; check out the full changelog to discover the full extent of this release.
+
+### Music playback
+
+Playing media is the key feature of Jellyfin. Up until now the Android TV app had many issues playing every possible format. The code
+was also hard to work with making it challenging to fix the issues or implement new features. To encounter this we decided to rewrite all
+playback code completely from scratch. In this release the initial results are ready for daily use and playing music will use the new
+playback code!
+
+While a lot of this work is not directly noticable when using the app, some of it is. Crashes should now be uncommon, starting music is
+faster and the remote control now responds more reliably. We've also added media sessions, this allows you to play/pause from other
+applications on your TV or even your smartphone!
+
+Improved codec detection should reduce transcoding and use direct play more often resulting in faster playback with less strain on the
+Jellyfin server. We already have plans to improve this even more and add preferences to fine-tune this for your specific devices. Other
+planned features include showing lyrics, enabling SyncPlay and support gapless playback.
+
+Work has already started to rework video playback based on this new code so we can add all kinds of new features with ease in a shared
+code base for both video and audio playback.
+
+![Screenshot of the screensaver during music playback in the Android TV app](./screensaver-music.png)
+
+### Screensaver
+
+The screensaver has received some nice additions. It was redesigned to be faster, with better animations and a much nicer appearance.
+Alongside this redesign, new functionality was added. Music playback has been added to the screensaver, replacing the old screensaver from
+the 'now playing' screen. Additionally, the screensaver is now enabled by default when the app is open, eliminating the need to enable it
+system-wide.
+
+And the best part: the screensaver now works when the video player is paused!
+
+![Screenshot of the screensaver in the Android TV app](./screensaver.png)
+
+### Search
+
+The search screen was partially rewritten to improve the performance. It will now cancel any pending search requests while typing and use
```

Comment:
```suggestion
The search screen was partially rewritten to improve performance. It will now cancel any pending search requests while typing and use
```